R-packages/evalcast/R/evaluate.R

```diff
@@ -111,51 +114,48 @@ evaluate_predictions <- function(
                               as_of = lubridate::as_date(Sys.Date()))
   score_card <- collapse_cards(score_card)
   score_card <- score_card %>%
-                 select(-c(quantile, value))
+    select(-.data$quantile, -.data$value)
   if (is.null(side_truth)) {
     score_card <- score_card %>%
-                    relocate(.data$ahead, .data$geo_value, .data$forecaster,
-                             .data$forecast_date, .data$data_source,
-                             .data$signal, .data$target_end_date,
-                             .data$incidence_period, .data$actual)
+      relocate(.data$ahead, .data$geo_value, .data$forecaster,
+               .data$forecast_date, .data$data_source,
+               .data$signal, .data$target_end_date,
+               .data$incidence_period, .data$actual)
   }
   score_card <- score_card %>%
                  relocate(attr(err_measures, "name"), .after = last_col())
   return(score_card)
 }
 
 get_covidcast_data <- function(predictions_cards,
-                               backfill_buffer) {
+                               backfill_buffer, 
+                               geo_type) {
   response <- predictions_cards %>%
-                select(.data$data_source, .data$signal) %>%
-                distinct()
+    select(.data$data_source, .data$signal) %>%
+    distinct()
   assert_that(nrow(response) == 1,
               msg = "All predictions cards should have the same response.")
   incidence_period <- unique(predictions_cards$incidence_period)
   assert_that(length(incidence_period) == 1,
               msg = "All predictions cards should have the same incidence
                      period.")
-  geo_type_len <- predictions_cards %>%
-    mutate(type_len = nchar(geo_value)) %>%
-    distinct(type_len)
-  assert_that(nrow(geo_type_len) == 1,
-              msg = "All predictions cards should have the same geo_type.")
-  geo_type <- ifelse(geo_type_len$type_len == 2L, "state", "county")
-
+  
+  geo_type <- geo_type_selector(geo_type, predictions_cards)
```

Comment:
This return value is never used so it should not be named.

Reply:
It gets used on line 165.

Reply:
Ah, my ctrl+f skipped that for whatever reason